src/MonthPicker.js

```diff
@@ -142,9 +142,12 @@ along with this program.  If not, see
     }
 
     function _makeDefaultButton(options) {
+        var span = $('<span />');
+        span.attr( 'id', 'MonthPicker_Button_' + this.id );
+        span.attr( 'class', 'month-picker-open-button' );
+        span.text( options.i18n.buttonText );
```

Comment:
At first I thought this can break users code if a server side template engine will HTML encode the translated button text.

But after some checking I found that template engines will only HTML encode unsafe characters.

Quoting [/lib/handlebars/utils.js](https://github.com/wycats/handlebars.js/blob/e8ceafa6525a5d41a20aa74dfe24bf99d0bc2fcd/lib/handlebars/utils.js#L1) from Handlebars.js runtime:
```
const escape = {
  '&': '&amp;',
  '<': '&lt;',
  '>': '&gt;',
  '"': '&quot;',
  "'": '&#x27;',
  '`': '&#x60;',
  '=': '&#x3D;'
};
```

@lucymtc I will merge this pull request if you can keep the HTML strings as is, and only set the id attribute and text (values that can come from user input in real edge cases) using the `.attr()` and `.text()` methods.

As I've said before I will post an update about why I left the HTML strings this way.